### DISCUSSION TITLE
Feature Request: Speaker Differentiation in Transcription

### DISCUSSION THREAD

**Discussion by Osso44** [Purfview/whisper-standalone-win#287] [General]
Hello Purfview,

I have a question about your program. If I transcribe a conversation with three people, does the program recognize who is speaking based on their voice and format it as a dialogue like "Speaker 1: ..., Speaker 2: ..., Speaker 3: ..."? If this feature is not available, is there a plan to include it in future updates?

Thank you in advance!

**Comment by thbaero** [comment]
Check #281 and consider donating even though the goal was already met :)

**Reply by Osso44** [reply]
I don't understand it, does the function I'm looking for exist or is it still to come?

**Reply by thbaero** [reply]
The function you are looking for is called "Speaker Diarization". It is not included in faster-whisper-xxl at this moment. But it might be available in the future.

@Purfview  was kind enough to offer implementing the feature if there are enough people supporting the effort through donation. That's why he created the call for donation (#281). The donation goal was already met (e.g. I donated 50£). However, I would appreciate if you could also check whether you would like to acknowledge the work of purfview through a donation from your side :) 

There is no information when the feature will be available. But I have trust in Purfview to implement the feature according to his availability. After all, this is a individually driven project - maybe even some sort of hobby of Purfview. I am grateful for his effort!

**Reply by Osso44** [reply]
Certainly, if the feature is implemented and works well, I am willing to make a donation as a recognition of his work.
